lerna.json

```diff
@@ -4,6 +4,7 @@
     "libs/*"
   ],
   "version": "independent",
+  "npmClient": "npm",
```

Comment:
this is the default, not needed (see https://lerna.js.org/docs/api-reference/configuration#npmclient)